src/pages/index.tsx

```diff
@@ -227,18 +213,14 @@ export default function Home() {
                   <div className="absolute group-hover:text-orange-600 text-2xl flex justify-center translate-y-32">
                     <BsChevronCompactDown />
                   </div>
-
-                  <div className="group-hover:text-orange-600 text-sm translate-y-4 text-gray-400">
-                    {content.text}
-                  </div>
-                </div>
-              </div>
-            );
-          })}
-        </div>
         <div className="flex flex-row flex-wrap justify-center">
           {walletDataSet.map((content, index) => (
-            <ProductCard key={index} content={content} />
+            <ProductCard
+              key={index}
+              content={content}
+              image={content.image}
+              imageAlt={content.imageAlt}
```

Comment:
This seems odd to me that the content is already passed down, but you're specifying image and imageAlt as content.image and content.imageAlt. Both of these already have been passed down through the content prop

Reply:
removed